File: DAL/Models/OnlineShoping/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models.OnlineShoping
{
    public class Order
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public Customer Customer { get; set; }
        public DateTime? OrderDate { get; set; }
        public DateTime? RequestDate { get; set; }
        public DateTime? DueDate { get; set; }
        public bool IsOpen { get; set; }
        public bool IsVisible { get; set; }
        public double Tax { get; set; }
        public double Discount { get; set; }
        public double TotalPrice { get; set; }
        public double TotalPriceAfterDiscount { get; set; }
        // Final Price
        public double TotalPriceAfterTax { get; set; }
        public ICollection<OrderItems> OrderItems { get; set; }
    }
}

File: DMS_Task/ClientApp/src/app/models/order.ts
import { OrderItem } from "./order-item";

export class Order {
    id?: number;
    fullName?: string;
    email?: string;
    orderDate?: Date;
    dueDate?: Date;
    tax?: number;
    discount?: number;
    totalPrice?: number;
    totalPriceAfterDiscount?: number;
    totalPriceAfterTax?: number;
    orderItems: OrderItem[]
}
